server/auth/config/passport.js

```diff
@@ -0,0 +1,43 @@
+const LocalStrategy = require('passport-local').Strategy
+// need a User model in db or something of equivalence (mock import)
+const { User } = require('../models')
+
+module.exports = function (passport) {
+  passport.use(
+    new LocalStrategy(
+      {
+        // by default, local strategy uses username, we will override with email
+        usernameField: 'email'
+      },
+      async (email, password, done) => {
+        try {
+          const user = await User.findOne({
+            where: { email: email.toLowerCase() }
+          })
+          if (!user) {
+            return done(null, false, { message: 'Email not found' })
+          }
+          const isMatch = await user.isValidPassword(password)
+          if (!isMatch) {
+            return done(null, false, { message: 'Incorrect password' })
+          }
+          return done(null, user)
+        } catch (err) {
+          return done(err)
+        }
+      }
+    )
+  )
+
+  passport.serializeUser((user, done) => {
+    done(null, user.id)
+  })
+  passport.deserializeUser(async (id, done) => {
+    try {
+      const user = await User.findByPk(id)
```

Comment:
I think you can just use `Admin.query().findById()` now.
https://vincit.github.io/objection.js/api/query-builder/find-methods.html#findbyid

Reply:
Cleaned up this code to use `.query().findById(id)` instead of `.findByPk(id)`